pkg/swarm/distance_test.go

```diff
@@ -0,0 +1,79 @@
+// Copyright 2020 The Swarm Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package swarm
+
+import (
+	"testing"
+)
+
+type distanceTest struct {
+	x      []byte
+	y      []byte
+	result string
+}
+
+type distanceCmpTest struct {
+	x      []byte
+	y      []byte
+	z      []byte
+	result int
+}
+
+var (
+	distanceTests = []distanceTest{
+		{
+			x:      MustParseHexAddress("9100000000000000000000000000000000000000000000000000000000000000").Bytes(),
+			y:      MustParseHexAddress("8200000000000000000000000000000000000000000000000000000000000000").Bytes(),
+			result: "8593944123082061379093159043613555660984881674403010612303492563087302590464",
+		},
+	}
+
+	distanceCmpTests = []distanceCmpTest{
+		{
+			x:      MustParseHexAddress("9100000000000000000000000000000000000000000000000000000000000000").Bytes(),
+			y:      MustParseHexAddress("8200000000000000000000000000000000000000000000000000000000000000").Bytes(),
+			z:      MustParseHexAddress("1200000000000000000000000000000000000000000000000000000000000000").Bytes(),
```

Comment:
Why not use same letters as in implementation for clarity? `z` -> `y` `a` -> `x` `x` -> `y`

Reply:
you tell me... its your implementation :D